rules/windows/powershell/powershell_module/posh_pm_invoke_obfuscation_clip.yml

```
@@ -7,9 +7,9 @@ status: experimental
 description: Detects Obfuscated use of Clip.exe to execute PowerShell
 references:
     - https://github.com/SigmaHQ/sigma/issues/1009  #(Task 26)
-author: Jonathan Cheong, oscd.community
+author: Jonathan Cheong, oscd.community, DTS_SOC
```

You don't become an author by fixing typos
```suggestion
author: Jonathan Cheong, oscd.community
```